api/apps/system_app.py

```
@@ -58,6 +59,27 @@ def run_migrations(current_user):
       return get_json_result(data="Migrations ran successfully.")
     except Exception as e:
       return server_error_response(e)
+        
+@manager.route("/reset-database", methods=["POST"])
```

Ruff (0.8.2)

Undefined name `manager`

(F821)